packages/aws-cdk-lib/core/lib/cfn-fn.ts

```diff
@@ -233,10 +233,12 @@ export class Fn {
   /**
    * The intrinsic function ``Fn::FindInMap`` returns the value corresponding to
    * keys in a two-level map that is declared in the Mappings section.
+   * Prefer to use `CfnMapping.findInMap`.
+   * Warning: do not use with lazy maps.
```

Comment:
Is this how we like to convey warnings / best practices?

Reply:
I think you should explain succinctly why you should not use with lazy maps